src/Sylius/Bundle/CoreBundle/Distributor/TaxesDistributor.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Bundle\CoreBundle\Distributor;
+
+/**
+ * @author Mateusz Zalewski <mateusz.zalewski@lakion.com>
+ */
+class TaxesDistributor implements TaxesDistributorInterface
```

Comment:
Is it really only taxes distributor? It is generic and can be used to distribute any number.